docs/tutorials/findoneandupdate.md

```diff
@@ -78,3 +86,31 @@ Here's what the `res` object from the above example looks like:
      age: 29 },
   ok: 1 }
 ```
+
+## Updating Discriminator Keys
+
+Mongoose prevents updating the [discriminator key](https://mongoosejs.com/docs/discriminators.html#discriminator-keys) using `findOneAndUpdate()` by default.
```

Comment:
```suggestion
Mongoose prevents updating the [discriminator key](../discriminators.html#discriminator-keys) using `findOneAndUpdate()` by default.
```

we should use relative urls, right?

Reply:
You're right. I thought for some reason we switched to absolute urls to allow clicking on links directly from GitHub markdown, but looks like that's not the case, we use relative URLs in markdown.